README.md

```diff
@@ -1,6 +1,6 @@
 # pkgnet <img src="man/figures/logo.png" align="right" alt="" width="120" />
 
-[![CRAN\_Status\_Badge](https://www.r-pkg.org/badges/version-last-release/pkgnet)](https://cran.r-project.org/package=pkgnet) [![CRAN\_Download\_Badge](https://cranlogs.r-pkg.org/badges/grand-total/pkgnet)](https://cran.r-project.org/package=pkgnet) [![Build Status](https://img.shields.io/travis/uptake/pkgnet.svg?label=build&logo=travis&branch=main)](https://travis-ci.org/uptake/pkgnet)
+[![CRAN\_Status\_Badge](https://www.r-pkg.org/badges/version-last-release/pkgnet)](https://cran.r-project.org/package=pkgnet) [![CRAN\_Download\_Badge](https://cranlogs.r-pkg.org/badges/grand-total/pkgnet)](https://cran.r-project.org/package=pkgnet) [![GitHub Actions Build Status](https://github.com/uptake/pkgnet/workflows/Tests/badge.svg?branch=main)](https://github.com/uptake/pkgnet/actions)
```

Comment:
You can do

```
https://github.com/uptake/pkgnet/actions?query=workflow%3ATests+branch%3Amain
```

so that the page you click to is filtered to the main branch tests.

Reply:
ooooooo cool!